waddrmgr/db.go

```diff
@@ -70,11 +71,31 @@ type addressType uint8
 
 // These constants define the various supported address types.
 const (
-	adtChain  addressType = 0 // not iota as they need to be stable for db
-	adtImport addressType = 1
-	adtScript addressType = 2
+	adtChain    addressType = 0 // not iota as they need to be stable for db
+	adtImport   addressType = 1
+	adtScript   addressType = 2
+	adtWatching addressType = 3
 )
 
+// falseByte and trueByte are consts used to serialize boolean values.
+const (
+	falseByte byte = iota
+	trueByte
+)
+
+// boolAsByte converts a bool to a byte.
+func boolAsByte(b bool) byte {
+	if b {
+		return trueByte
+	}
+	return falseByte
+}
+
+// byteAsBool converts a byte to a bool.
+func byteAsBool(b byte) bool {
+	return b != 0
```

Comment:
Not that I find 0 hard to read here, but just above you've used constants for `trueByte` and `falseByte`, so shouldn't this be consistent with `b != falseByte`?